linkerd/app/outbound/src/tls/logical/route/filters.rs

```diff
@@ -10,11 +10,11 @@ where
     let filters: &[tls::Filter] = &t.param();
     if let Some(filter) = filters.iter().next() {
         match filter {
-            tls::Filter::ForbiddenRoute => {
+            tls::Filter::Forbidden => {
                 return Err(errors::TLSForbiddenRoute.into());
             }
 
-            tls::Filter::InvalidBackend(message) => {
+            tls::Filter::Invalid(message) => {
                 return Err(errors::TLSInvalidBackend(message.clone()).into());
```

Comment:
We can change the name of these error types to be more generic if we want to. We currently return Invalid only if backends are invalid and Forbidden only if a route is forbidden, so this seems fine as it is for now 🤷